terasoluna-gfw-functionaltest-parent/pom.xml

```diff
@@ -28,6 +28,27 @@
         <name>TERASOLUNA Framework Team</name>
         <url>http://terasoluna.org</url>
     </organization>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <artifactId>maven-war-plugin</artifactId>
+                    <configuration>

```

Comment:
@btyoshikawaa
OK. We regard ``parent`` as a manager of whole ``terasoluna-gfw-functionaltest``.
Anyway we've defined other pathes of ``env`` such as configs...